site/_layouts/doc.liquid

```diff
@@ -196,9 +196,52 @@
                   <a href="#" class="sub-menu"><i class="fal fa-fw fa-rss"></i> Events & Webhooks <i class="fal fa-chevron-{% if page.url contains "/events-webhooks/" %}up{% else %}down{% endif %} fa-fw"></i></a>
                   <ul>
                     <li {% if page.url == "/docs/v1/tech/events-webhooks/" %}class="active"{% endif %}><a href="/docs/v1/tech/events-webhooks/" >Overview</a></li>
-                    <li {% if page.url == "/docs/v1/tech/events-webhooks/events/" %}class="active"{% endif %}><a href="/docs/v1/tech/events-webhooks/events" >Events</a></li>
                     <li {% if page.url == "/docs/v1/tech/events-webhooks/writing-a-webhook/" %}class="active"{% endif %}><a href="/docs/v1/tech/events-webhooks/writing-a-webhook" >Writing a Webhook</a></li>
                     <li {% if page.url == "/docs/v1/tech/events-webhooks/securing/" %}class="active"{% endif %}><a href="/docs/v1/tech/events-webhooks/securing" >Securing Webhooks</a></li>
+                    <li {% if page.url contains "/events-webhooks/event-bodies/" %} class="open"{% endif %}>
+                      <a href="#" class="sub-menu"> Events <i class="fal fa-chevron-{% if page.url contains "/events-webhooks/events/" %}up{% else %}down{% endif %} fa-fw"></i></a>
+                      <ul>
+                        <li {% if page.url == "/docs/v1/tech/events-webhooks/events" %}class="active"{% endif %}><a href="/docs/v1/tech/events-webhooks/events" >Overview</a></li>
+                        <li {% if page.url == "/docs/v1/tech/events-webhooks/event-bodies/audit-log-create/" %}class="active"{% endif %}><a href="/docs/v1/tech/events-webhooks/event-bodies/audit-log-create/">Audit Log Create</a></li>
+                        <li {% if page.url == "/docs/v1/tech/events-webhooks/event-bodies/event-log-create/" %}class="active"{% endif %}><a href="/docs/v1/tech/events-webhooks/event-bodies/event-log-create/">Event Log Create</a></li>
+                        <li {% if page.url == "/docs/v1/tech/events-webhooks/event-bodies/jwt-public-key-update/" %}class="active"{% endif %}><a href="/docs/v1/tech/events-webhooks/event-bodies/jwt-public-key-update/">JWT Public Key Update</a></li>
+                        <li {% if page.url == "/docs/v1/tech/events-webhooks/event-bodies/jwt-refresh/" %}class="active"{% endif %}><a href="/docs/v1/tech/events-webhooks/event-bodies/jwt-refresh/">JWT Refresh</a></li>
```

Comment:
I wonder if we can use a liquid data object or something to just iterate over an list of items to build this out? May be easier to maintain. @mooreds may know how to do this, or you can google it.

Reply:
I did this here with the theme API: https://github.com/FusionAuth/fusionauth-site/pull/886 

Big fan of the `data` capabilities of jekyll.

Reply:
I was going to merge, but didn't get a chance to review this comment.  I'll hold off until tomorrow AM.  Otherwise, this PR is CLOSE